lib/include/hse_ikvdb/kvs_cparams.h

```diff
@@ -36,6 +36,12 @@ kvs_cparams_get(
     size_t                    buf_sz,
     size_t *                  needed_sz);
 
+merr_t
+kvs_cparams_set(
```

Comment:
This function isn't really necessary since you cannot set a cparam. Can remove if you all think it is worthless. It was essentially free to implement.

Reply:
Agreed, it's worthless..